### ISSUE TITLE
CONNECTION_TIMED_OUT (-118)

### ISSUE THREAD

**Issue by SlientTime** [minbrowser/min#2344]
If i use VPN to visit Google, Duckduckgo and youtube, the browser shows me the error-CONNECTION_TIMED_OUT (-118). How to fix it?

**Comment by LvQingRui** [comment]
I had the same problem

**Comment by Kepler-69c** [comment]
Can you try the following things:

- Can these pages be opened with other browsers when the VPN is on?
- Can Min open the pages when the VPN is switched off?

**Comment by bdubaut** [comment]
> Can these pages be opened with other browsers when the VPN is on?

Yes, I have the same issue, and I can confirm that this is working fine on Chrome & Safari.

> Can Min open the pages when the VPN is switched off?

No, because that is the purpose of the VPN, to. keep internal pages unaccessible unless connected to the VPN.

**Comment by enokoneko** [comment]
Same question here.
When I set proxy rules to http port on min preferences, I got `CONNECTION_CLOSED (-100)`,
When I set to socks5 port, I got `EMPTY_RESPONSE (-324)`.
Safari and Firefox working fine.
---
min 1.33.1
macOS 15.0.1
